src/app/tools/symmetries.cpp

```diff
@@ -40,14 +41,19 @@ void HorizontalSymmetry::generateStrokes(const Stroke& mainStroke, Strokes& stro
   Stroke stroke2;
   for (const auto& pt : mainStroke) {
     Stroke::Pt pt2 = pt;
+    if (dynamics->isDynamic()) {
+      brushSize = pt2.size;
+      brushCenter = (brushSize - brushSize % 2) / 2;
```

Comment:
It looks like the brush center (brush->center()) is lost here, but I might be wrong. Maybe some tests could be created about this in https://github.com/aseprite/tests/blob/c4252301bf0272433d9c22d8922b536e963f602c/scripts/tools.lua#L391

Reply:
I ran the tests and it was OK. To test a pen with dynamic size, we need first an emulation of pressure in app.useTool. I think, may be in a future.